pysb/util.py

```diff
@@ -9,7 +9,13 @@
 
 
 def alias_model_components(model=None):
-    """Make all model components visible as symbols in the caller's global namespace"""
+    """
```

Comment:
This ought to generate a `DeprecationWarning`, e.g.
```python
warnings.warn("alias_model_components is deprecated, please use Model.alias_components",
              DeprecationWarning, stacklevel=2)
```
Python does silence that warning by default but it does seem to be the proper thing to do.

Also the docstring should use a sphinx deprecation notice rather than plain text:
```rst
.. deprecated:: 1.2
   Use :func:`pysb.core.Model.alias_components` instead.
```